app/invites/server/functions/useInviteToken.js

```diff
@@ -18,7 +18,11 @@ export const useInviteToken = (userId, token) => {
 	const user = Users.findOneById(userId);
 	Users.updateInviteToken(user._id, token);
 
-	Invites.increaseUsageById(inviteData._id);
+	const subscription = Subscriptions.findOneByRoomIdAndUserId(room._id, user._id, { fields: { _id: 1 } });
+	const userInRoom = subscription != null;
+	if (!userInRoom) {
```

Comment:
you don't need a const for this, just do !subscription. Thanks for the help

Reply:
Made the changes